.github/workflows/analyze.yml

```
@@ -0,0 +1,28 @@
+name: Analyze
+
+on: [push, pull_request]
+
+jobs:
+  dart:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Find changed packages
+        run: |
+          CHANGED_PACKAGES=$(python3 tools/run_command.py plugins --run-on-changed-packages --base-sha $(git rev-parse HEAD^))
+          if [[ ! -z $CHANGED_PACKAGES ]]; then
+              echo $CHANGED_PACKAGES
+              echo "HAS_CHANGED_PACKAGES=true" >> "$GITHUB_ENV"
+          fi
+      - name: Install flutter-tizen
+        if: ${{ env.HAS_CHANGED_PACKAGES == 'true' }}
+        run: |
+          git clone https://github.com/flutter-tizen/flutter-tizen.git
+      - name: Analyze changed packages
+        if: ${{ env.HAS_CHANGED_PACKAGES == 'true' }}
+        run: |
+          ROOT_DIR=`pwd`
+          export PATH=$PATH:`pwd`/flutter-tizen/bin
```

```suggestion
          echo "$GITHUB_WORKSPACE/flutter-tizen/bin" >> $GITHUB_PATH
```
You can use environment files instead. 
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files

---

Thank you for your review. I pushed a new commit which is simpler than before